src/npr/ont_pipeline.py

```diff
@@ -239,6 +261,10 @@ def read_flowcell_info(config, info_dict, base_path):
                     print(head[0].index('barcode_kit'))
         else:
             sys.exit("no json file, no final summary txt file found. exiting.")
+        ##################################################################################
```

Comment:
?

Reply:
those were some notes marks, a further discussion about if we parse the text file for the minimal information capture. Removed